lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -31,20 +31,19 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       PROVIDER_SPECIFIC_FIELDS = {
         'ovirt' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"])
+          Fields::Field.new(:label => _('UUID'), :path => [:uuid])
```

Comment:
Compared code between api version v1 and v2
```
#foreman/app/views/api/v1/compute_resources/ovirt.json.rabl
attributes :user, :uuid

#foreman/app/views/api/v2/compute_resources/ovirt.json.rabl
attributes :user, :datacenter
```
```
#foreman/app/views/api/v1/compute_resources/vmware.json.rabl
attributes :user, :uuid, :server

#foreman/app/views/api/v2/compute_resources/vmware.json.rabl
attributes :user, :datacenter, :server, :set_console_password
```
It looks like they intentionally moved from uuid to datacener in v2. Any thoughts? shall i open new PR to add uuid?